### PR TITLE
Add translations for profile settings

### PR DESCRIPTION
**Pull request by pixeldesu**
Cleans up/Improves the locales for Profile Settings

**Testing:**
* Look at `/settings/profile` and verify it still works and the texts display correctly

### CODE REVIEW

File: app/views/settings/_profile.haml [hunk 1]
```diff
@@ -6,46 +6,45 @@
...
 
       - %i[profile_picture_x profile_picture_y profile_picture_w profile_picture_h].each do |attrib|
         = f.hidden_field attrib, id: attrib
 
       - %i[profile_header_x profile_header_y profile_header_w profile_header_h].each do |attrib|
         = f.hidden_field attrib, id: attrib
 
-      = f.submit t('views.actions.save'), class: 'btn btn-primary'
-
+      = f.primary
```
Comment (raccube) [inline app/views/settings/_profile.haml:35]: What does this render out to?

**Review by raccube — COMMENTED** [review]
Looks good, just unsure about one thing.

**Review by raccube — APPROVED** [review]

File: config/locales/activerecord.en.yml [hunk 2]
```diff
@@ -11,8 +11,26 @@ en:
...
+      profile:
+        motivation_header: "Shown in the header of the question box on your profile. Motivate users to ask you questions!"
       services/twitter:
         post_tag: "Automatically append a tag to your shared answers. A # symbol is not automatically prepended."
+  helpers:
+    submit:
+      user:
+        update: :voc.save
+      profile:
+        update: :voc.save
```
Comment (nilsding) [inline config/locales/activerecord.en.yml]: Stimky dog sez: Final newline missing

**Review by nilsding — CHANGES_REQUESTED** [review]

File: config/locales/activerecord.en.yml (hunk 2 above)
Comment (pixeldesu) [reply config/locales/activerecord.en.yml]: Hounding

**Review by nilsding — APPROVED** [review]
Stimky dog sez: it smells too good.  Yerf!

**Pull request merged**